src/manifest.rs

```diff
@@ -275,6 +303,14 @@ impl FromStr for RawManifest {
 /// Map representation of the dependency list
 pub type DependencyMap = HashMap<PackageName, DependencyManifest>;
 
+/// A manifest can define either a package or a workspace
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ManifestType {
+    /// The Manifest defines a package
+    Package,
+    /// The Manifest defines a workspace
+    Workspace,
+}
```

Comment:
```suggestion
}

```

Nit: Missing newline